test/osmodifiertest/pyproject.toml

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+[tool.isort]
+line_length = 120
+profile = "black"
+skip = ["build", "venv", ".venv", "out", "__pycache__", ".pytest_cache", ".mypy_cache"]
+extend_skip = ["build", "venv"]
+skip_glob = ["build/*", "venv/*", ".venv/*"]
+
+[tool.black]
+line-length = 120
+extend-exclude = '''
```

Comment:
In `vmtests`, there are two directories named `vmtests`. The second `vmtests` directory contains the actual Python code. Please follow that same pattern here to avoid needing to add all these excludes and skips,